### PR TITLE
update to v2.0.4

### PR DESCRIPTION
**Pull request by Skellers21**
update to v2.0.4

also includes daemon graphic and alignment for windows node

### CODE REVIEW

File: depends/packages/openssl.mk [hunk 1]
```diff
@@ -92,7 +92,7 @@ $(package)_config_opts_i686_mingw32=mingw
 endef
 
 define $(package)_preprocess_cmds
-  sed -i.old "/define DATE/d" util/mkbuildinf.pl && \
+  sed -i.old 's/built on: $$$$date/built on: date not available/' util/mkbuildinf.pl && \
```
Comment (hTrap) [inline depends/packages/openssl.mk]: typo?
$$$$date

**Comment by hTrap** [comment]
Rename contrib/debian/examples/zcash.conf
to zclassic.conf

File: README.md [hunk 2]
```diff
@@ -1,4 +1,4 @@
-ZClassic 2.0.3
+Zcash 2.0.4
```
Comment (hTrap) [inline README.md]: Why is it zcash and not zclassic 2.0.4?

File: doc/man/zcash-cli.1 [hunk 3]
```diff
@@ -1,9 +1,9 @@
 .\" DO NOT MODIFY THIS FILE!  It was generated by help2man 1.47.6.
```
Comment (hTrap) [inline doc/man/zcash-cli.1]: Shouldn't we update this to zclassic-cli

File: README.md (hunk 2 above)
Comment (hTrap) [inline README.md]: Why is it ZCash 2.0.4 and not ZClassic 2.0.4

File: doc/man/zcash-tx.1 [hunk 4]
```diff
@@ -1,9 +1,9 @@
 .\" DO NOT MODIFY THIS FILE!  It was generated by help2man 1.47.6.
```
Comment (hTrap) [inline doc/man/zcash-tx.1]: Shouldn't we update this to zclassic-tx

File: doc/man/zcashd.1 [hunk 5]
```diff
@@ -1,9 +1,9 @@
 .\" DO NOT MODIFY THIS FILE!  It was generated by help2man 1.47.6.
```
Comment (hTrap) [inline doc/man/zcashd.1]: Shouldn't we update this to zclassicd

**Review by hTrap — COMMENTED** [review]
contrib/debian/examples/zcash.conf  to be renamed as contrib/debian/examples/zclassic.conf
as mentioned in src/bitcoind.cpp

File: doc/man/zcash-tx.1 (hunk 4 above)
Comment (Skellers21) [reply doc/man/zcash-tx.1]: updated

File: doc/man/zcashd.1 (hunk 5 above)
Comment (Skellers21) [reply doc/man/zcashd.1]: updated

File: README.md (hunk 2 above)
Comment (Skellers21) [reply README.md]: updated

File: doc/man/zcash-cli.1 (hunk 3 above)
Comment (Skellers21) [reply doc/man/zcash-cli.1]: updated

**Review by zclass — APPROVED** [review]
looks good

File: doc/man/zcashd.1 (hunk 5 above)
Comment (hTrap) [reply doc/man/zcashd.1]: shouldn't the filename be updated as well?

**Pull request merged**